ruby/tests/api-test.cc

```diff
@@ -49,6 +49,11 @@ TEST_CASE("api::is_*", "[ruby-api]") {
 
         REQUIRE(ruby.is_fixednum(ruby.eval("2")));
         REQUIRE_FALSE(ruby.is_fixednum(ruby.eval("1.5")));
+		REQUIRE(ruby.is_fixednum(ruby.eval("12345678912")));
```

Comment:
That number is a bit big to be a fixednum on some Ruby versions on Windows. I would test something like UINT_MAX instead.